src/include/miopen/solver.hpp

```diff
@@ -4561,7 +4561,7 @@ struct PerformanceConfigHipImplicitGemmFwdXdlops
     PerformanceConfigHipImplicitGemmFwdXdlops() : PerformanceConfigHipImplicitGemmFwdXdlops(0, "")
     {
     }
-    PerformanceConfigHipImplicitGemmFwdXdlops(bool)
+    explicit PerformanceConfigHipImplicitGemmFwdXdlops(bool)
```

Comment:
Any custom defined constructor that takes a single parameter must be marked `explicit` to avoid automatic conversion. For example, without `explicit`, if I pass `bool` var to a `fun(const PerformanceConfigXXX& obj)`, `C++` will silently create an object along the way instead of throwing an error. `explicit` causes the compiler to throw an error.

Reply:
[Resolved]